### ISSUE TITLE
The react-popover library, used in the color sliders, is unmaintained

### ISSUE THREAD

**Issue by adroitwhiz** [LLK/scratch-paint#1122]
### Expected Behavior

We should use libraries that still receive updates

### Actual Behavior

`react-popover`, used for the color slider popover, was last updated 2 years ago.

It contains a [bug where it recognizes mouse and touch events but ignores pointer events](https://github.com/littlebits/react-popover/blob/905e7545e32806abf8ee38f4557621e9bc5b1c1a/source/index.js#L434-L435), which is blocking https://github.com/LLK/paper.js/pull/37. I was hoping to get a fix merged into it, but they don't seem to be looking at any pull requests.

**Comment by Jacco** [comment]
This one seems to have about the same functionality and is still maintained:

https://www.npmjs.com/package/react-tiny-popover

You could also consider rolling a self-made version to have one less dependancy.

**Comment by adroitwhiz** [comment]
I looked into `react-tiny-popover` but it uses `createRef` which isn't supported in the old version of React that Scratch currently uses.

**Comment by Jacco** [comment]
Yikes ok, version 16 is used right?

**Comment by adroitwhiz** [comment]
Yes, but there are a variety of point-releases that add new features and deprecate (but don't break) old ones.

**Comment by Jacco** [comment]
[popperjs](https://popper.js.org/)

As far as I can see you have to do a bit of work yourself but seems good.

**Comment by Jacco** [comment]
Ah also 16.8.0

**Comment by fsih** [comment]
We should look at updating our React version
